neuraxle/data_container.py

```diff
@@ -142,24 +142,40 @@ def hash_summary(self):
             m.update(str.encode(str(current_id)))
         return m.hexdigest()
 
-    def convolved_1d(self, stride, kernel_size) -> Iterable['DataContainer']:
+    def convolved_1d(self, stride, kernel_size, include_incomplete_pass: bool = True, default_value: Any = None) -> Iterable['DataContainer']:
```

Comment:
It is very very rare that we won't include an incomplete pass. The arg is ok. 

However, what I see that is missing is the case when, for a batch size of 350 for instance, the first 100 would be good values, and the last 250 would be using default values. 
==> We would need in that case an option to NOT use any default value, and to dynamically reduce the batch_size in those cases. 

I think that this could be a null object (design pattern) passed to the `default_value: Any = None` argument. So it would be an `default_value: Union[Any, AbsentValuesNullObject] = None` type. Passing a `AbsentValuesNullObject` would momentarily make the last batch be of just 100 rather than 350. 

That is my only comment for this PR. I would like to see it done. The rest looks good. 

Hopefully this will fix all our problems for un-full batches in the future.

Reply:
what do you mean by null object ? you want a new abstraction that we can use later in the pipeline to know that this is an incomplete batch ? I don't get it